module/tests/test_constants.py

```diff
@@ -0,0 +1,93 @@
+# Test cases
+ADHD = {
+    'ID': 'Q181923',
+    'Name': 'attention deficit hyperactivity disorder',
+    'Link': 'https://www.wikidata.org/wiki/Q181923',
+    'DrugIDs': ['Q410441', 'Q419008', 'Q423288', 'Q1706418', 
+                'Q898407', 'Q1207210', 'Q58396', 'Q834280',
+                'Q412221', 'Q402633', 'Q191924', 'Q179452', 
+                'Q417240', 'Q61387', 'Q1706418', 'Q2506823',
+                'Q5613599', 'Q1207210', 'Q419008','Q422112'],
+    'DrugNames': ['modafinil', 'pemoline', 'desipramine', 'dextroamphetamine', 
+                  'venlafaxine', 'dexmethylphenidate', 'imipramine', 'bupropion',
+                  'clonidine', '(±)-deprenyl', 'D-methamphetamine', 'DL-amphetamine',
+                  'atomoxetine', 'nortriptyline', 'dextroamphetamine', '(R)-amphetamine',
+                  'guanfacine', 'dexmethylphenidate', 'pemoline', 'methylphenidate'],
+    'FDADrugIDs': ['Q1706418', 'Q2506823', 'Q5613599', 'Q1207210', 
+                   'Q419008','Q422112'],
+    'FDADrugNames': ['dextroamphetamine', '(R)-amphetamine', 'guanfacine', 'dexmethylphenidate', 
+                     'pemoline', 'methylphenidate']
+}
+
+DEPRESSION = {
+    'ID': 'Q42844',  
+    'Name': 'major depressive disorders',
+    'Link': 'https://www.wikidata.org/wiki/Q42844',
+    'DrugIDs':['Q2622367', 'Q2273909', 'Q834280', 'Q238544',
+               'Q395229', 'Q58356', 'Q6535779', 'Q27139037', 
+               'Q418361', 'Q411932', 'Q1452256', 'Q243547', 
+               'Q334477', 'Q208144'],
+    'DrugNames': ['Mood stabilizer', 'serotonin–norepinephrine reuptake inhibitor', 'bupropion', 'oxitriptan'
+                  'agomelatine', 'amoxapine', 'levomilnacipran', 'vilazodone hydrochloride', 
+                  'maprotiline', 'duloxetine', 'levosulpiride', 'ketamine',
+                  'selective serotonin reuptake inhibitor', 'antipsychotic'], 
+    'FDADrugIDs':['Q834280', 'Q238544', 'Q395229', 'Q58356', 
+                  'Q6535779', 'Q27139037', 'Q418361', 'Q411932', 
+                  'Q1452256'],
+    'FDADrugNames': ['bupropion', 'oxitriptan','agomelatine', 'amoxapine', 
+                     'levomilnacipran', 'vilazodone hydrochloride', 'maprotiline', 'duloxetine', 
+                     'levosulpiride']
+}
+
+DIABETES = {
+    'ID': 'Q3025883',  
+    'Name': 'type-2 diabetes',
+    'Link': 'https://www.wikidata.org/wiki/Q3025883',
+    'DrugIDs':['Q27275212', 'Q772735', 'Q409898', 'Q15269678',
+               'Q899036', 'Q27077223', 'Q15269682', 'Q954845'
+               'Q19484', 'Q288280'],
+    'DrugNames': ['luseogliflozin', 'miglitol', 'dapagliflozin', 'teneligliptin'
+                  'Colesevelam', 'ertugliflozin', 'anagliptin', 'statin',
```

Comment:
Does capitalization matter? Should we convert everything to lowercase when comparing entities?

Reply:
These are the names I got from WikiData, I wanted to keep their format consistent. We use the IDs for comparison anyways.

Reply:
Gotcha. Thanks!